dom.go

```diff
@@ -516,9 +519,7 @@ func (h *HTML) reconcileChildren(prev *HTML) (pendingMounts []Mounter) {
 		}
 		h.removeChildren(prevChildren)
 		return pendingMounts
-	}
-
-	if len(prev.children) > len(h.children) {
+	} else if len(prev.children) > len(h.children) {
```

Comment:
This seemed to be an oversight / bug. See the commit message for better details / a code example, but I think the worst outcome from this would be that it would invoke `unmount` twice potentially. I am not 100% sure about that though.

Regardless, I think this is more correct behavior. Let me know if you think otherwise!

Reply:
There's a return statement in the preceding block, so no need for `else`?

Reply:
😮 I feel dumb. You're 100% right. I will revert.